apps/src/code-studio/viewAsRedux.js

```diff
@@ -13,12 +13,18 @@ export const SET_VIEW_TYPE = 'viewAs/SET_VIEW_TYPE';
 
 export default function reducer(state = ViewType.Student, action) {
   if (action.type === SET_VIEW_TYPE) {
-    const viewType = action.viewType;
-    if (!ViewType[viewType]) {
+    let viewType = action.viewType;
+    if (viewType === 'Instructor') {
```

Comment:
(minor) It's probably useful to include a comment here to indicate that this is a temporary change, why it's there, and when it's safe to remove.  Just in case it lives for longer than we originally intended.  :-)

Reply:
Thanks for always helping to make our codebase easier to use by making sure we include documentation!